omf-python/tests/test_number_attribute.py

```diff
@@ -53,22 +62,28 @@ def test_should_return_discrete_colormap_boundaries(self) -> None:
         )
         actual_boundaries = self.reader.array_boundaries(boundary_array)
 
-        expected_boundaries = [946688400.0, 946692000.0, 946695600.0]
-        self.assertEqual(actual_boundaries, expected_boundaries)
+        # TODO: check these values
+        expected_boundaries = numpy.array(
+            [946688400000000, 946692000000000, 946695600000000]
+        )
```

Comment:
Also need to determine why the type‐aware `array_boundaries()` returns different values.

Reply:
I think it’s because these are datetimes that were being returned with microseconds precision rather than seconds precision.

Reply:
Ideally this would return an array with dtype "datetime64[us]", but that might be a pain from Rust.

Reply:
Currently I’ve got it returning tuples with [datetime](https://docs.python.org/3/library/datetime.html) objects.